rb_ws/src/buggy/scripts/auton/path_planner.py

```diff
@@ -40,31 +47,39 @@ def __init__(self, nominal_traj:Trajectory) -> None:
         self.debug_grid_cost_publisher = rospy.Publisher(
             "/auton/debug/grid_cost", Float64, queue_size=0
         )
+
+        self.other_buggy_xtrack_publisher = rospy.Publisher(
+            "/auton/debug/other_buggy_xtrack", Float64, queue_size=1
+        )
+
         self.nominal_traj = nominal_traj
+        self.left_curb = left_curb
 
         # TODO: estimate this value based on the curvature of NAND's recent positions
         self.other_steering_angle = 0
 
+    def offset_func(self, dist):
+        return self.OFFSET_SCALE_CROSS_TRACK / \
```

Comment:
imo its clearer if  `PathPlanner.OFFSET_SCALE_CROSS_TRACK` is used. But its a nitpick, no need to change if you don't want.

Reply:
Also function header